site/source/docs/optimizing/Optimizing-Code.rst

```diff
@@ -221,6 +221,15 @@ Enable :ref:`debugging-EMCC_DEBUG` to output files for each compilation phase, i
 
 .. _optimizing-code-unsafe-optimisations:
 
+Allocation
+----------
+
+The default ``malloc/free`` implementation used is ``dlmalloc``. You can also
+pick ``emmalloc`` (``-sMALLOC=emmalloc``) which is smaller but less fast, or
+``mimalloc`` (``-sMALLOC=mimalloc``) which is larger but scales better in a
+multithreaded application with contention on ``malloc/free`` (see
+:ref:`Allocator_performance`).
```

Comment:
I guess we should consider making it the default for threaded builds onces we have enough trust in it?

Reply:
Given the code size and memory overhead, I'm not sure. Maybe. Perhaps if we can make it smaller and leaner (like fixing that emmalloc alignment issue https://github.com/emscripten-core/emscripten/issues/20645).